src/appleseed.shared/application/progresstilecallback.cpp

```diff
@@ -80,12 +80,19 @@ namespace
 
         void on_tiled_frame_begin(const Frame* frame) override
         {
-            m_stopwatch.start();
+            // Do not restart the stopwatch if it is already running.
+            if (m_rendered_pixels == 0.0)
```

Comment:
`m_rendered_pixels` is an integral type.

Reply:
🤦‍♂️ Sorry!